### PR TITLE
Minor typos corrections

### PR DESCRIPTION
**Pull request by OzTK**
First and foremost, thank you @hendricius for your incredible work, you're my number 1 resource to upskill with bread baking ❤️ 

I noticed a few typos and duplications as I was reading the book and thought I'd fix them.

### CODE REVIEW

File: book/history/sourdough-history.tex [hunk 1]
```diff
@@ -21,17 +21,6 @@ \chapter{The history of sourdough}%
 the evening of December~31.  It seems that humans managed to arrive just
 in time to join the celebration at the end of the year.
 
-The story of sourdough bread begins in ancient oceans. These oceans were the
```
Comment (OzTK) [inline book/history/sourdough-history.tex:24]: Not 100% sure that's the section that should be removed, I tried to infer from the look of it. If you can confirm @hendricius

**Comment by cedounet** [comment]
thanks @OzTK , I will take some of your commits in my branches
https://github.com/hendricius/the-sourdough-framework/tree/bread-types 
and
https://github.com/hendricius/the-sourdough-framework/tree/wheat-chapter

For the repetition one, The boss will decide how he wants it, does happen when you add a section that there is some overlap with existing text elsewhere...

**Comment by hendricius** [comment]
@OzTK - sweet - thanks for the nice words and the PR. That looks like a mistake indeed and can be improved. Since @cedounet has taken some of your commits into his branch I'll do the same and open up a PR. There's going to be more rework of the chapter incoming soon.

**Comment by hendricius** [comment]
@OzTK sorry for dissecting your commit here into multiple branches. For some reason I could not cherry pick from your remote, so I opened up a new PR #294. I will close this for now since @cedounet took some of your work to the other branches. Thanks again!

**Comment by hendricius** [comment]
Ah thanks ❤️

**Pull request closed without merging**